src/vl-upload.js

```diff
@@ -384,4 +402,23 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
   _enableAutoProcessQueue() {
     this._dropzone.options.autoProcessQueue = true;
   }
+
+  _addSlots() {
+    Promise.all([this.uploadElement]).then(() => {
+      const subTitleSlotNodes = this._subTitleSlotElement.assignedNodes();
+      if (subTitleSlotNodes.length > 0) {
+        const small = this.uploadElement.querySelector('small');
```

Comment:
Dat is dan een element dat de achterliggende JS code genereert?

Reply:
in die <small></small> wordt de vertaling van de subtitle gezet via het i18n systeem. Ik neem hier het slot indien aanwezig en plaats wat meegegeven wordt in die small